courses/free-courses-en.md

```diff
@@ -497,6 +497,7 @@
 ### Cuda
 
 * [CUDA Crash Course](https://www.youtube.com/playlist?list=PLxNPSjHT5qvtYRVdNN1yDcdSl39uHV_sU) - CoffeeBeforeArch
+* [CUDA Programming Course – High-Performance Computing with GPUs (12 hours)](https://youtu.be/86FAWCzIe_4?si=nNR5hvJKqT2-M2m9) - Elliot Arledge (FreeCodeCamp)
```

Comment:
We do not allow link shorteners. 
See docs/CONTRIBUTING.md#platforms-and-access-notes
No need to add `(12 hours)` to the item, that should go in your PR description to help the reviewers.

Reply:
I apologize, I didn't notice that copying a link from YouTube shortens the link.
I added the full link now, thanks for being patient with me!